lib/db_connection/connection.ex

```diff
@@ -93,6 +93,11 @@ defmodule DBConnection.Connection do
   def init({mod, opts, mode, info}) do
     queue  = if mode == :sojourn, do: :broker, else: :queue.new()
     broker = if mode == :sojourn, do: info
+
+    if owner = Keyword.get(opts, :owner) do
+      Process.link(owner)
```

Comment:
Ah yes, forgot about it. In case of crashes, should we just exit with the same reason as in the DOWN message?